static/_redirects

```diff
@@ -1,3 +1,7 @@
+# 10/16/2023
+
+/nodes/run/with-installer /nodes/run/with-installer/preparing-your-environment
+
```

Comment:
I usually add redirects at the bottom of the file to keep the changes in a chronological order. I see that some changes from march were included at the top here before we joined, so we should probably move those as well.

Reply:
please move this to the bottom of the `_redirects` file